docs/topics/con_server-sent-events.adoc

```diff
@@ -0,0 +1,9 @@
+= Server-Sent Events
+
+Server-sent events (SSE) is a protocol that you can use to push unidirectional events form an event source to a web-based or browser-based client over HTTP.
```

Comment:
Server-sent events is a protocol that pushes unidirectional events from an event source to a web-based or browser-based client over HTTP.